src/ripple/app/ledger/Ledger.cpp

```diff
@@ -416,7 +416,7 @@ Ledger::setAccepted(NetClock::time_point closeTime,
 bool Ledger::addSLE (SLE const& sle)
 {
     SHAMapItem item (sle.getIndex(), sle.getSerializer());
-    return stateMap_->addItem(item, false, false);
+    return stateMap_->addItem(std::move(item), false, false);
```

Comment:
Would it make more sense to skip instantiating the variable entirely, so the compiler can optimize without needing to be told about the move? i.e.
```
bool Ledger::addSLE (SLE const& sle)
{
    return stateMap_->addItem(SHAMapItem (sle.getIndex(), sle.getSerializer()), false, false);
}
```